pdr_backend/pdr_dashboard/util/db.py

```diff
@@ -374,6 +428,40 @@ def predictoors_metrics(self):
             "Gross Income": tot_gross_income,
         }
 
+    def get_first_and_last_slot_timestamp(self):
+        first_timestamp, last_timestamp = self._query_db(
+            f"""
+                SELECT 
+                    MIN(timestamp) as min,
+                    MAX(timestamp) as max
+                FROM 
+                    {Slot.get_lake_table_name()}
+            """,
+            scalar=True,
+        )
+        return first_timestamp / 1000, last_timestamp / 1000
+
+    def get_feeds_data(self):
```

Comment:
I would name these "refresh_..._data" maybe? Otherwise it looks like it should return something. And if these refreshes happen simultaneously, I would do both blocks in the same function